test/e2e-v2/cases/mysql/prometheus-mysql-exporter/docker-compose.yml

```
@@ -0,0 +1,61 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+version: '2.1'
+
+services:
+  oap:
+    extends:
+      file: ../../../script/docker-compose/base-compose.yml
+      service: oap
+    environment:
+      SW_OTEL_RECEIVER: default
+      SW_OTEL_RECEIVER_ENABLED_OC_RULES: mysql
+    ports:
+      - 12800
+  mysql:
+    image: mysql:5.7
+    networks:
+      - e2e
+    ports:
+      - "3306:3306"
+    environment:
+      - MYSQL_ROOT_PASSWORD=password
```

Maybe you can add a container here and mount a test sql file and execute the commands inside it. (Not completed and tested, don't simply commit the suggested codes)

```suggestion
  mysql:
    image: mysql:5.7
    networks:
      - e2e
    environment:
      - MYSQL_ROOT_PASSWORD=password
      
  mysql-load:
    image: mysql:5.7
    networks:
      - e2e
    entrypoint: bash
    command:
      - -c
      - "mysql -h mysql -u root -p password test < test.sql"

```